### PR TITLE
Enhancement/donate button

### PR DESCRIPTION
**Pull request by Irsanarisandy**
### DoD (Definition of Done) Checklist

- [x] User Documentation written _(remove if not applicable)_
- [x] There are no outstanding high priority defects or regressions
- [x] Feature is tested in agreed devices and latest version of browsers**
- [x] The developer is confident that Acceptance Criteria have been met
- [x] Branch rebased against `develop`
- [x] Non-MVP enhancements have been captured in another card
- [x] Add `ngrok` link to site

##### * Design DOD
- [x] The design signed off by Beth (if appropriate)*
- [x] Accurately matches design unless there is a good reason not to
- [x] Hover states & interactions are correct
- [x] No regressions of other styles

##### ** Agreed Browsers and Devices
- **Browsers** - Chrome (Development Team), IE, Firefox, Safari
- **Devices** - iPad Pro, Surface Tablet.

### Peer Review Checklist
- [x] Must be peer reviewed by one of the team members first before passing it onto mentor
- [x] Manually tested
- [x] Runs without issue locally
- [x] Browser/Device tested
- [x] Code is quality reviewed
- [x] Feedback is provided and documented
- [x] Matches all Acceptance Criteria

### CODE REVIEW

File: themes/app/templates/Includes/Header.ss [hunk 1]
```diff
@@ -30,8 +30,8 @@
                 <% end_loop %>
             </div>
         </div>
-        <div slot="button" class="main-nav__donate">
-            <button class="btn btn-outline-secondary" type="button">DONATE NOW</button>
+        <div slot="button" class="main-nav__button">
+            <a class="btn btn-outline-secondary" href="$SiteConfig.HeaderButtonLink.Link" role="button">$SiteConfig.HeaderButtonText</a>
```
Comment (IshaqIdris) [inline themes/app/templates/Includes/Header.ss:34]: Nice! I like this line, might have to steal it for myself 😉

Comment (Irsanarisandy) [reply themes/app/templates/Includes/Header.ss:34]: 😘

File: app/_config/app.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 Silverstripe\SiteConfig\SiteConfig:
   extensions:
-    - App\extensions\FooterConfig
+    - App\Extensions\ExtraConfig
```
Comment (pjayme) [inline app/_config/app.yml]: Its convention to suffix filenames with `Extension` and have some form of relevance to the config you are extending

I would rename this to `SiteConfigExtension` :) 

Ensure you update the classes below and all other references

File: app/code/extensions/ExtraConfig.php [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\TreeDropdownField;
 use SilverStripe\AssetAdmin\Forms\UploadField;
-use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
 
-class FooterConfig extends DataExtension
+class ExtraConfig extends DataExtension
 {
-
     private static $db = [
```
Comment (pjayme) [inline app/code/extensions/ExtraConfig.php]: Don't forget your docblocks for all the fields and functions in this class

File: themes/app/src/scss/_variables.scss [hunk 4]
```diff
@@ -1,8 +1,6 @@
 /**
  * Application variables
  */
-
-$contentPadding: 20px 40px;
```
Comment (pjayme) [inline themes/app/src/scss/_variables.scss:5]: why was this removed?

File: themes/app/src/scss/components/_navigation.scss [hunk 5]
```diff
@@ -72,11 +72,11 @@ $verticalGap: 15px;
     }
 }
 
-.main-nav__donate {
+.main-nav__button {
```
Comment (pjayme) [inline themes/app/src/scss/components/_navigation.scss:75]: nice refactor here :) good case of where `donate` is too specific and it modifiable text hence the use of `button` is good here

File: themes/app/src/scss/_variables.scss (hunk 4 above)
Comment (Irsanarisandy) [reply themes/app/src/scss/_variables.scss:5]: Cause it's no longer used, since using Bootstrap.

Comment (pjayme) [reply themes/app/src/scss/_variables.scss:5]: 👍
